pkg/virt-controller/network/pod_annotations_test.go

```diff
@@ -123,7 +126,20 @@ var _ = Describe("pod annotations", func() {
 				downwardapi.NetworkInfoAnnot: `{"interfaces":[{"network":"foo","deviceInfo":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.2"}}}]}`,
 			}))
 		})
-		It("should have both network-pci-map, network-info entries when there is SRIOV interface and binding plugin interface with device-info", func() {
+		It("should have network-info entry when there is one SR-IOV interface", func() {
+			networks := []virtv1.Network{
+				*libvmi.MultusNetwork("foo", "default/sriov"),
+			}
+
+			interfaces := []virtv1.Interface{
+				libvmi.InterfaceDeviceWithSRIOVBinding("foo"),
+			}
+			podAnnotationMap := network.GeneratePodAnnotations(networks, interfaces, networkStatus, bindingPlugins)
+			Expect(podAnnotationMap).To(Equal(map[string]string{
+				downwardapi.NetworkInfoAnnot: `{"interfaces":[{"network":"foo","deviceInfo":{"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:65:00.2"}}}]}`,
+			}))
+		})
+		It("should have network-info entry when there is SR-IOV interface and binding plugin interface with device-info", func() {
```

Comment:
Since the interface filtering was moved to the `GeneratePodAnnotations` method please add more networks - non binding plugin, binding plugin with no device info to show those are filtered out.

Reply:
Done